.github/workflows/version-packages.yml

```diff
@@ -38,5 +38,7 @@ jobs:
       - name: Create or update Version Packages PR
         if: ${{ steps.has-changesets.outputs.has-changesets == 'true' }}
         uses: changesets/action@b98cec97583b917ff1dc6179dd4d230d3e439894
+        with:
+          version: yarn version:latest
```

Comment:
This will run our custom versioning command with `yarn generateVersions`. This will ensure "Version Packages" PR will update `version.ts` when it bumps up the version.